dotCMS/src/main/webapp/html/js/tinymce/js/tinymce/plugins/doteditimage/langs/en.js

```diff
@@ -0,0 +1,3 @@
+tinyMCE.addI18n("en", {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid trailing commas in object or array literals](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5494719)